runtime/core/cmake/openvino.cmake

```diff
@@ -0,0 +1,89 @@
+      set(URL_HASH "SHA256=93d6bd23d5e8306384bdc8f3746fe56baad1a0f87b97d0b384b4c0b158227d76")
+    elseif(LSB_RELEASE_ID_SHORT STREQUAL "CentOS" AND LSB_RELEASE_VERSION_SHORT VERSION_LESS 8.10)
+      set(VINO_URL "https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.3/linux/l_openvino_toolkit_rhel8_${VINO_VERSION}.9052.9752fafe8eb_x86_64.tgz")
+      set(URL_HASH "SHA256=519cbdf95d73489c771357f27dbc5829057fcc3b23fd89be06e290b41a8ff934")
+    elseif(LSB_RELEASE_ID_SHORT STREQUAL "Debian" AND LSB_RELEASE_VERSION_SHORT VERSION_LESS_EQUAL 9.13 AND X86)
+      set(VINO_URL "https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.3/linux/l_openvino_toolkit_debian9_${VINO_VERSION}.9052.9752fafe8eb_arm64.tgz")
+      set(URL_HASH "SHA256=4099bb9a257af46fbf7d5c30414ec4e308542704eac666b4b213362208bf8b16")
+    else()
+      message(STATUS "Linux OS without exist binary, compile OpenVINO source.")
+    endif()
+  elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin" AND X86)
+    set(VINO_URL "https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.3/macos/m_openvino_toolkit_macos_10_15_${VINO_VERSION}.9052.9752fafe8eb_x86_64.tgz")
+    set(URL_HASH "SHA256=235221536ddde7937db9ba6cd3c262d6220d2e7c7a24438d046fcd3ca0c73bed")
+  else()
+    message(FATAL_ERROR "Unsupported CMake System Name '${CMAKE_SYSTEM_NAME}' (expected 'Windows', 'Linux' or 'Darwin')")
+    set(VINO_URL "")
+    set(URL_HASH "")
+    endif()
+
+  if(${VINO_URL} STREQUAL "")
+    set(ENABLE_INTEL_GPU OFF CACHE BOOL "Build OV with GPU" FORCE)
+    set(ENABLE_INTEL_MYRIAD OFF CACHE BOOL "Build OV with MYRIAD" FORCE)
+    set(ENABLE_INTEL_GNA OFF CACHE BOOL "Build OV with GNA" FORCE)
+    set(ENABLE_INTEL_CPU ON CACHE BOOL "Build OV with CPU" FORCE)
+    set(BUILD_SHARED_LIBS ON CACHE BOOL "Build OV in shared library" FORCE)
+    set(ENABLE_SAMPLES OFF CACHE BOOL "Build OV with Example" FORCE)
+    set(ENABLE_TESTS OFF CACHE BOOL "Build OV with Tests" FORCE)
+    set(ENABLE_COMPILE_TOOL OFF CACHE BOOL "Build OV with COMPILE_TOOL" FORCE)
+    set(ENABLE_PYTHON OFF CACHE BOOL "Build python" FORCE)
+    set(THREADING "TBB")
+    #set(CMAKE_BUILD_TYPE "Debug")
+    FetchContent_Declare(openvino
+      GIT_REPOSITORY https://github.com/openvinotoolkit/openvino.git
+      GIT_TAG 9752fafe8ebf7e30dfea7edd447ff3bf0ac1d01d) # 2022.3.0 on Dec 20, 2022
+    FetchContent_MakeAvailable(openvino)
+  else()
+    FetchContent_Declare(openvino
+      URL ${VINO_URL}
+      URL_HASH ${URL_HASH}
+    )
+    FetchContent_MakeAvailable(openvino)
+    include_directories(${openvino_SOURCE_DIR}/include)
+    link_directories(${openvino_SOURCE_DIR}/lib)
+  endif()
+
+  if(MSVC)
+    file(GLOB ONNX_DLLS "${openvino_SOURCE_DIR}/lib/*.dll")
+    file(COPY ${VINO_DLLS} DESTINATION ${CMAKE_BINARY_DIR}/bin/${CMAKE_BUILD_TYPE})
+  endif()
```

Comment:
Q1:

line84, `ONNX_DLLS` ? I think it should be `VINO_DLLS`.

Besides, after downloading the Windows version of openvino, I find that `*.dll` are located in `runtime/bin/intel64/release/*.dll`, not `xxx/lib/*.dll`, see:

![image](https://user-images.githubusercontent.com/13466943/220538955-ebe45176-e14b-41ad-91e8-e96e07dc6614.png)


Q2:

Do we really need to copy those dlls ? or line84:line86 are just copied from onnx.cmake without justification ?

Reply:
I shall remove, cause directly provide pck for Win*, no need to self-compile OV source.